.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: python
+
+# sudo false implies containerized builds
+sudo: false
+
+addons:
+  postgresql: "9.6"
+  apt:
+    packages:
+    - postgresql-9.6
```

Comment:
Travis CI will work with PostgreSQL v9.6 or later. README will be updated to reflect this

Reply:
Good to note. Yes, please update the README accordingly.